.env

```diff
@@ -1,3 +1,13 @@
+REACT_APP_API_URI=
+REACT_APP_APPLICATION_NAME=$npm_package_name
 REACT_APP_OIDC_AUTHORITY="https://api.hel.fi/sso/openid/"
+REACT_APP_OIDC_CLIENT_ID="https://api.hel.fi/auth/omahelsinki"
 REACT_APP_PROFILE_AUDIENCE="https://api.hel.fi/auth/profiles"
+REACT_APP_PROFILE_GRAPHQL=
 REACT_APP_OIDC_SCOPE="openid profile $REACT_APP_PROFILE_AUDIENCE"
+REACT_APP_SENTRY_DSN=
+REACT_APP_VERSION=$npm_package_version
+#TRANSLATION_LANGUAGES=en,fi,sv
+#TRANSLATIONS_URL=https://docs.google.com/spreadsheets/d/18dpV7aA-t_6zMRUO5hP7DMfx0q4hdu-AlLpRjkB_0zI/gviz/
+#TRANSLATION_PROJECT_NAME=
+#SASS_PATH=./src/assets/styles
```

Comment:
We are not using SASS for omahessu, so this could be deleted.

Reply:
Removed it